docs/platform/features/group-chat.mdx

```diff
@@ -0,0 +1,327 @@
+
+Retrieve memories from a specific participant in a group chat:
+
+<CodeGroup>
+
+```python Python
+# Get memories for a specific participant
+filters = {
+    "AND": [
+        {"user_id": "charlie"},
+        {"run_id": "GroupChat_1"}
+    ]
+}
+
+charlie_memories = client.get_all(version="v2", filters=filters, page=1)
+print(charlie_memories)
+```
+
+```json Output
+[
+    {
+        "id": "147559a8-c5f7-44d0-9418-91f53f7a89a4",
+        "memory": "suggests considering Angular because it has great enterprise support",
+        "user_id": "charlie",
+        "session_id": "GroupChat_1",
+        "metadata": null,
+        "categories": null,
+        "created_at": "2025-06-21T05:51:11.007223-07:00",
+        "updated_at": "2025-06-21T05:51:11.626562-07:00",
+        "expiration_date": null,
+        "structured_attributes": null,
+        "internal_metadata": null,
+        "deleted_at": null
+    }
+]
+```
+
+</CodeGroup>
+
+### Search Within Group Chat Context
+
+Search for specific information within a group chat session:
+
+<CodeGroup>
+
+```python Python
+# Search within group chat context
+filters = {
+    "AND": [
+        {"user_id": "charlie"},
+        {"run_id": "GroupChat_1"}
+    ]
+}
+
+search_response = client.search(
+    query="What are the tasks?",
+    filters=filters,
+    version="v2"
+)
+print(search_response)
+```
+
+```json Output
+[
+    {
+        "id": "147559a8-c5f7-44d0-9418-91f53f7a89a4",
+        "memory": "suggests considering Angular because it has great enterprise support",
+        "user_id": "charlie",
+        "session_id": "GroupChat_1",
+        "metadata": null,
+        "categories": null,
+        "created_at": "2025-06-21T05:51:11.007223-07:00",
+        "updated_at": "2025-06-21T05:51:11.626562-07:00",
+        "expiration_date": null,
+        "structured_attributes": null,
+        "internal_metadata": null,
+        "deleted_at": null,
+        "score": 0.45
+    }
+]
+```
+
+</CodeGroup>
+
+## Async Mode Support
+
+Group chat also supports async processing for improved performance:
+
+<CodeGroup>
+
+```python Python
+# Group chat with async mode
+response = client.add(
+    messages,
+    run_id="GroupChat_Async",
+    output_format="v1.1",
+    infer=True,
+    async_mode=True
+)
+print(response)
+```
+
+</CodeGroup>
+
+## Message Format Requirements
+
+### Required Fields
+
+Each message in a group chat must include:
+
+- `role`: The participant's role (`"user"`, `"assistant"`, `"agent"`)
+- `content`: The message content
+- `name`: The participant's name (required for group chat detection)
+
+### Example Message Structure
+
+```json
+{
+  "role": "user",
+  "name": "Alice",
+  "content": "I think we should use React for the frontend"
+}
+```
+
+### Supported Roles
+
+- **`user`**: Human participants (memories stored with `user_id`)
+- **`assistant`**: AI assistants (memories stored with `agent_id`)
+- **`agent`**: AI agents (memories stored with `agent_id`)
```

Comment:
What is the difference between assistant and agent here? I dont think we need the `agent`